pystarport/cluster.py

```diff
@@ -906,6 +908,20 @@ def init_cluster(
 ):
     config = yaml.safe_load(open(config_path))
 
+    if "dotenv" in config:
+        dotenv = config.pop("dotenv", {})
+        if not isinstance(dotenv, str):
+            raise ValueError(f"Invalid value passed to dotenv: {dotenv}")
+        config_vars = load_system_envvars()
+        env_path = Path(config_path).parent.joinpath(dotenv)
```

Comment:
`dotenv` could also be an absolute path, need to handle that I think.

Reply:
Ok, let me modify a bit.

Reply:
I tested, this works with absolute path as well.